compiler/AST/AstToText.cpp

```diff
@@ -1607,21 +1607,32 @@ void AstToText::appendExpr(CallExpr* expr, bool printingType, const char *outer,
   }
 }
 
+void AstToText::appendExprTypeVar(DefExpr* expr, bool isType)
+{
+  if (isType) {
```

Comment:
This check is a duplicate of the check prior to calling the function.  I don't think it or the isType argument are needed

Reply:
I have modified the function as you suggested.